ansible/requirements.yml

```diff
@@ -125,8 +125,8 @@
 - src: openmicroscopy.reboot-server
   version: 0.1.0
 
-- src: git+https://github.com/openmicroscopy/ansible-role-devspace.git
-  version: master
+- src: git+https://github.com/jburel/ansible-role-devspace.git
```

Comment:
Bump before merge?

Reply:
yes

Reply:
@joshmoore ok to merge the role PR so i can bump it